### PR TITLE
Documentation Improvement

### PR DESCRIPTION
**Pull request by Naveen7003**
In this contribution, I focused on improving the clarity and readability of the project's documentation. This involved updating various texts and revising paragraphs to fix typos, improve structure, and enhance the overall flow. My changes ensure a more polished and user-friendly documentation experience, which is crucial for both new and experienced contributors.

### CODE REVIEW

**Comment by Naveen7003** [comment]
@shhossain Could you please add the following labels to this pull request? 
- kind/documentation
- size/S

File: README.md [hunk 1]
```diff
@@ -65,17 +65,17 @@ Computer science is the study of computers and computing and their theoretical a
...
 
 ### Common Boolean Operators
 | Operator | Name |               Description               |
 | :------: | :--: | :-------------------------------------: |
-|    !     | NOT  |    Negates the value of the operand.    |
-|    &&    | AND  | Returns **true** if both operands are true. |
-|   \|\|   |  OR  | Returns **true** if one or all operands are true. |
+|    !     | NOT  | Negates (inverts) the value of the operand.   |
+|    &&    | AND  | Returns true if both operands are true. |
+|    ||    |  OR  | Returns true if at least one operand is true. |
```
Comment (shhossain) [inline README.md:78]: `\` is used as escape charecter. so it's needed.

**Review by shhossain — CHANGES_REQUESTED** [review]
Change requested.

**Comment by Naveen7003** [comment]
I have fixed the mistake.

**Review by shhossain — APPROVED** [review]
Nice work.

**Comment by shhossain** [comment]
@all-contributors add @Naveen7003 for doc

**Pull request merged**